packages/tx-construction/test/tx-builder/mocks.ts

```diff
@@ -0,0 +1,12 @@
+import { Cardano } from '@cardano-sdk/core';
```

Comment:
I think we can import the similar mock defined in input-selection/test/util and avoid redefining here